server/plugins/auth-utils.js

```diff
@@ -76,6 +76,9 @@ async function authUtilsPlugin(fastify) {
   fastify.decorate("prepareOidcLoginRedirect", (request, oidcConfig, authorizationEndpoint) => {
     request.log.info("Preparing OIDC login redirect.");
 
+    const { redirectTo } = request.query;
+    request.session.set("postLoginRedirectRoute", redirectTo);
```

Comment:
we might want to validate that the redirect is a relative url or an absolute url on our domain?

Reply:
I added a check to allow only for relative redirect paths (starting with `/` or `#`)